apps/coscrad-frontend/src/components/footer/footer.spec.tsx

```diff
@@ -39,4 +39,41 @@ describe('Footer', () => {
 
         expect(screenRes).toBeTruthy();
     });
+
+    it('renders email icon when email is given', () => {
+        const email = 'johndoe@johndoe.com';
+        const myConfigurableContent = { ...dummyConfigurableContent, email };
+
+        renderWithProviders(
+            <MemoryRouter>
+                <Footer />
+            </MemoryRouter>,
+            {
+                contentConfig: myConfigurableContent,
+            }
+        );
+
+        const searchPattern = new RegExp(email);
+        const screenRes = screen.getByText(searchPattern);
+
+        expect(screenRes).toBeTruthy();
```

Comment:
Also check that the icon is rendered. You might need to put a `data-testid` on said ICON.

Reply:
It works now, thank you.